person.rb

```diff
@@ -27,4 +28,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental unless @rentals.include?(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Kindly update this method to like:
```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
as it's the same relation as between `Rental` and `Book`.